### PR TITLE
Random 7.1 fixes.

### PR DESCRIPTION
**Pull request by awgil**

### CODE REVIEW

File: FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs [hunk 1]
```diff
@@ -9,30 +9,41 @@ namespace FFXIVClientStructs.FFXIV.Client.Game.Character;
...
-    [FieldOffset(0x680)] public MountContainer Mount;
-    [FieldOffset(0x6E8)] public CompanionContainer CompanionData;
-    [FieldOffset(0x708)] public DrawDataContainer DrawData;
-    [FieldOffset(0x8E8)] public OrnamentContainer OrnamentData;
-    [FieldOffset(0x960)] public ReaperShroudContainer ReaperShroud;
+    [FieldOffset(0x620)] public EmoteController EmoteController;
+    [FieldOffset(0x660)] public MountContainer Mount;
+    [FieldOffset(0x6C8)] public CompanionContainer CompanionData;
+    [FieldOffset(0x6E8)] public DrawDataContainer DrawData;
+    [FieldOffset(0x8D8)] public byte FreeCompanyCrestBitfield; // & 0x01 for offhand weapon, & 0x02 for head, & 0x04 for top, ..., & 0x20 for feet
```
[the remark this answers is not in the document]
Comment (awgil) [reply FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs:19]: This is very likely (why else would it move), but I'm not familiar with DrawData stuff tbh...

[the remark this answers is not in the document]
Comment (awgil) [reply FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs]: Updated the DrawData size and moved bitfield there.

**Comment by awgil** [comment]
Note: I've updated UIState.Buddy offset, but did not yet touch other fields - i suspect anything beyond PlayerState might be shifted

**Comment by Haselnussbomber** [comment]
Yes, a couple structs inside UIState after PlayerState got bigger.

**Pull request merged**